### ISSUE TITLE
Getting an Error with PostalCodeElement

### ISSUE THREAD

**Issue by chobo2** [stripe/react-stripe-elements#369]
Hi

I am trying to create my own form using react elements.

```
import React, { Component } from 'react';
import { injectStripe } from 'react-stripe-elements';
import { CardNumberElement, CardExpiryElement, CardCVCElement, PostalCodeElement } from 'react-stripe-elements';


class PaymentFormComponent extends Component {
  constructor(props) {
    super(props);
  }

  render() {
    return (
      <div className="payment-form-container">
        <h3 className="title is-3 payment-header">Payment For Invoice:</h3>
        <div className="field name-on-card-container">
          <label className="label">Name on Card</label>
          <div className="control">
            <input className="input is-large" type="text" onChange={this.onNameChange} />
          </div>
        </div>
        <div className="field card-number-container">
          <label className="label">Card Number</label>
          <div className="control has-icons-left">
            <CardNumberElement className="input is-large" />
            <span className="icon is-left">{/* <i className={cardTypeState} /> */}</span>
          </div>
        </div>
        <div className="field expiry-date-container">
          <label className="label">Expiry Date</label>
          <div className="control">
            <CardExpiryElement className="input is-large" />
          </div>
        </div>
        <div className="field cvc-container">
          <label className="label">CVC</label>
          <div className="control">
            <CardCVCElement className="input is-large" />
          </div>
        </div>
        <PostalCodeElement  />
         <div className="field cvc-container">
          <label className="label">Postal Code / Zip Code</label>
          <div className="control">
            <PostalCodeElement  />
          </div>
        </div> 
      </div>
    );
  }
}

export default injectStripe(PaymentFormComponent);
```
but I get these error

```
Warning: React.createElement: type is invalid -- expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.

Check the render method of `PaymentFormComponent`.
    in PaymentFormComponent (created by inject-PaymentFormComponent-with-domainStores-uiStores-routingStore)
    in inject-PaymentFormComponent-with-domainStores-uiStores-routingStore (created by InjectStripe(inject-PaymentFormComponent-with-domainStores-uiStores-routingStore))
    in InjectStripe(inject-PaymentFormComponent-with-domainStores-uiStores-routingStore) (created by CheckoutComponent)
    in Elements (created by CheckoutComponent)
    in div (created by CheckoutComponent)
    in CheckoutComponent (created by inject-CheckoutComponent-with-domainStores-uiStores-routingStore)
    in inject-CheckoutComponent-with-domainStores-uiStores-routingStore (created by Route)
    in Route (created by NavigationAreaComponent)
    in Switch (created by NavigationAreaComponent)
    in div (created by NavigationAreaComponent)
    in div (created by NavigationAreaComponent)
    in NavigationAreaComponent (created by inject-NavigationAreaComponent-with-domainStores-uiStores-routingStore)
    in inject-NavigationAreaComponent-with-domainStores-uiStores-routingStore (created by Route)
    in Route (created by withRouter(inject-NavigationAreaComponent-with-domainStores-uiStores-routingStore))
    in withRouter(inject-NavigationAreaComponent-with-domainStores-uiStores-routingStore) (created by Route)
    in Route (created by App)
    in Switch (created by App)
    in App (created by inject-App-with-domainStores-uiStores)
    in inject-App-with-domainStores-uiStores (created by Route)
    in Route (created by withRouter(inject-App-with-domainStores-uiStores))
    in withRouter(inject-App-with-domainStores-uiStores)
    in Provider
    in ErrorBoundary
    in Router
    in Provider


```
I am using  "react-stripe-elements": "^3.0.0",

**Comment by dweedon-stripe** [comment]
Hi @chobo2,

We removed `PostalCodeElement` in v3.0.0. We suggest that you build your own
postal code input and pass the postal in when using `createPaymentMethod`, `handleCardPayment`, `createToken`, etc.

**Comment by chobo2** [comment]
@dweedon-stripe 

Why does it seem like it is still accessible when I try to import it

```
import {
  CardNumberElement,
  CardExpiryElement,
  CardCVCElement,
  PostalCodeElement
} from 'react-stripe-elements';
```

Also why was it removed?

**Comment by dweedon-stripe** [comment]
Not sure what you mean. The export is undefined, which is why you see the error. Are you using Typescript? It looks like the third party typings, which we do not own or maintain, have not been updated to reflect the changes in v3.0.0

As to why it was removed: We removed it because there are a lot of corner cases in postal code validation. We determined that, given these issues that these corner cases created, it made more sense for users to collect postal code information with their own inputs where they would have more control over validation logic.
